pkg/connector/tickets.go

```diff
@@ -499,6 +503,7 @@ func (d *Connector) CreateTicket(ctx context.Context, ticket *v2.Ticket, schema
 				continue
 			}
 
+			l.Info("adding custom field to new ticket", zap.String("id", id), zap.Any("value", metaFieldValue))
```

Comment:
Maybe you may want to log this in the level of `DEBUG`? if not, please ignore this comment.

Reply:
I would like the customer to be able to see this value, it will help them to be able to configure their Jira Datacenter create ticket rules to see how C1 creates tickets. I think it is therefore informational and not necessarily debug information.